csrc/serde/fusion_record.cpp

```diff
@@ -812,6 +812,8 @@ void RecordFunctorFactory::setupFunctionMaps() {
   NVFUSER_BINARY_TV_OP("bitwise_and", bitwise_and)
   NVFUSER_BINARY_TV_OP("bitwise_or", bitwise_or)
   NVFUSER_BINARY_TV_OP("bitwise_xor", bitwise_xor)
+  NVFUSER_BINARY_TV_OP("logical_and", logical_and)
+  NVFUSER_BINARY_TV_OP("logical_or", logical_or)
```

Comment:
Does the absence of these mean that previously we could not serialize `logical_and` and `logical_or`?